components/Forms/Radio.tsx

```diff
@@ -36,54 +36,47 @@ export const Radio: React.VFC<InputProps> = ({
   return (
     <div className="radio">
       <fieldset>
-        {/* <QuestionLabel
+        <QuestionLabel
           name={name}
           type="radio"
           label={label}
           requiredText={requiredText}
           helpText={helpText}
-        /> */}
-        <legend
```

Comment:
not sure IF not having a legend would pass a11y

Reply:
I tried on the nu checker, it didn't complain

Reply:
@MarcoGoC  we already have a legend for each card, I don't think we need legend for each radio